var/spack/repos/builtin/packages/espresso/package.py

```diff
@@ -69,7 +77,6 @@ def check_variants(self, spec):
             raise RuntimeError(error.format(variant='elpa'))
 
     def install(self, spec, prefix):
```

Comment:
It seems the maintainers of Quantum Espresso fixed `make install` at some point, and now we are installing in:
```console
<prefix>/bin/bin
```
Can you please also take care of that?